DependencyInjection/Configuration.php

```diff
@@ -63,7 +63,7 @@
  *   - [max_files]: files to keep, defaults to zero (infinite)
  *   - [level]: level name or int value, defaults to DEBUG
  *   - [bubble]: bool, defaults to true
- *   - [file_permission]: string|null, defaults to null
+ *   - [file_permission]: int|null, defaults to null (0644)
```

Comment:
```suggestion
 *   - [file_permission]: int|null, defaults to null (0o644)
```

We need to use the proper octal notation if using octal for the default value